components/clp-py-utils/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "clp-py-utils"
+version = "0.1.0"
+description = "Utilities for other Python packages in CLP."
+authors = ["YScope Inc. <dev@yscope.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8 || ^3.10"
+mariadb = "~1.0.7"
```

Comment:
Did you check whether each component needs all these dependencies? E.g., does `clp-py-utils` require `mariadb` and `mysql-connector-python`?

Reply:
Yeah, sql_adapter.py needs these two